.circleci/config.yml

```diff
@@ -26,8 +26,8 @@ jobs:
             - run:
                 name: Install dependencies (dash)
                 command: |
-                    git clone --depth 1 git@github.com:plotly/dash.git dash-main
-                    git clone --depth 1 git@github.com:plotly/dash-html-components.git
+                    git clone --depth 1 https://github.com/plotly/dash.git dash-main
+                    git clone --depth 1 https://github.com/plotly/dash-html-components.git
```

Comment:
Why is https better than ssh?

Reply:
if you use `circleci cli`, ssh will always ask to confirm `yes/no`, as the main purpose is to get the source code, so https is slightly more automatic than ssh